bin/node/runtime/src/lib.rs

```diff
@@ -549,6 +555,16 @@ impl pallet_election_provider_multi_phase::Config for Runtime {
 	type MinerMaxWeight = MinerMaxWeight;
 	type MinerMaxLength = MinerMaxLength;
 	type MinerTxPriority = MultiPhaseUnsignedPriority;
+	type SignedMaxSubmissions = SignedMaxSubmissions;
+	type SignedRewardBase = SignedRewardBase;
+	type SignedRewardFactor = (); // no score-based reward
```

Comment:
I think it is safe to remove this for now. Giving reward based on how much a solution was improved is pretty flawed. You can submit a shitty solution yourself, and then improve it by a ton, and you are a banker now?

maybe check the logic. If this kind of gaming is allowed, we don't want it.

Reply:
61b76800b